src/source-control/GitSourceControl.ts

```diff
@@ -1,9 +1,31 @@
-import type { Commit, CommitHash } from "../shared/types";
+import { func } from "prop-types";
+import type { BranchName, Commit, CommitHash } from "../shared/types";
 import type {
   SourceControl,
   SourceControlRepositoryUpdateListener,
 } from "./SourceControl";
 
+var randomWords = require("random-words");
```

Comment:
We should be using imports

```suggestion
import randomWords from "random-words";

```